### DISCUSSION TITLE
Add Timeline settings similar to Desktop, to ignore name changes, joins etc

### DISCUSSION THREAD

**Discussion by moodler** [element-hq/element-meta#1856] [Ideas]
In our company people often change their "display name" in Matrix to reflect their status, particularly when they are on leave.

Unfortunately this clutters up the displays of chat rooms quite a lot, because there is a status update IN EVERY ROOM that person is in.   This makes the room listing kinda pointless, and hides the most useful information (the last messages posted)

![IMG_0327](https://github.com/vector-im/element-x-ios/assets/88039935/08414003-00fc-434f-8e89-10983200b46a)

I believe it obviously makes sense to add Timeline settings (just like Element on desktop) to ignore such changes, as well as join/leave and so on.

**Comment by kittykat** [comment]
Moving to Element Meta because this is a cross-platform issue (should behave the same way on EX Android and iOS)

**Comment by VolkerJunginger** [comment]
Thanks for reporting. We will look into a solution.
